helm-chart/templates/_config.tpl

```diff
@@ -31,7 +49,12 @@ control-plane {
         }
       }
       {{- end }}
-      debug.keep-load-generator-alive = {{ toJson .keepLoadGeneratorAlive }}
+      {{- with .extraContent }}
+      {{- range $key, $value := . }}
+      {{ $key }} = {{ toJson $value }}
+      {{- end }}
+      {{- end }}
```

Comment:
should we keep this?

Reply:
I kept it here since you added it to AWS Terraform configs for dev purposes. Will you need it in the future?